app/repositories/PathManager.scala

```diff
@@ -12,13 +12,13 @@ case class PathRemoveFailed(m: String) extends RuntimeException(m)
 
 object PathManager {
 
-  val httpClient = new OkHttpClient()
+  val httpClient = new OkHttpClient.Builder().connectTimeout(5, TimeUnit.SECONDS).build
```

Comment:
why do you need to override the default timeout  will be useful in a comment

Reply:
I was just keeping the existing timeout used by this class

Reply:
oh I see